src/core/server/utils/auth_info.ts

```diff
@@ -31,7 +32,9 @@ export const getPrincipalsFromRequest = (
     if (authState?.authInfo?.backend_roles) {
       payload[PrincipalType.Groups] = authState.authInfo.backend_roles;
     }
-    if (authState?.authInfo?.user_name) {
+    if (authState?.authInfo?.user_id) {
```

Comment:
This seems unrelated, is it by accident?

Reply:
Nope, as the change is minor so I want to include the change into in this PR as well. No need to raise a new PR for such 1 line change.